File: app/src/main/java/com/imooc/music/zzg/activity/MainActivity.java
package com.imooc.music.zzg.activity;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.imooc.music.zzg.BaseActivity;
import com.imooc.music.zzg.R;

public class MainActivity extends BaseActivity {

    @Override
    protected int getLayoutRes() {
        return R.layout.activity_main;
    }

    @Override
    protected void initView() {
      initNav(false,"慕课音乐",true);
    }

    @Override
    protected void initEvent() {

    }


}

File: app/src/main/java/com/imooc/music/zzg/activity/MainActivity.kt
package com.imooc.music.zzg.activity

import android.support.v7.widget.GridLayoutManager
import com.imooc.music.zzg.BaseActivity
import com.imooc.music.zzg.R
import com.imooc.music.zzg.adapters.MusicGridAdapter
import kotlinx.android.synthetic.main.activity_main.*


/**
 *Created by 18041at 2019/6/6
 *Function:
 */

class MainActivity:BaseActivity() {
    private  var musicAdapter:MusicGridAdapter?=null

    override fun getLayoutRes(): Int {
        return R.layout.activity_main
    }

    override fun initView() {
        initNav(false, "慕课音乐", true)
        musicAdapter=MusicGridAdapter(this)
        rv_grid.layoutManager=GridLayoutManager(this,3)
        rv_grid.adapter=musicAdapter
     }

    override fun initEvent() {

     }
}